modules/ROOT/pages/install-odatav2-tooling.adoc

```diff
@@ -0,0 +1,24 @@
+= Installing OData V2 Tooling
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To install APIkit for OData v2 plugin in Anypoint Studio:
```

Comment:
```suggestion
To install the APIkit for OData v2 plugin in Anypoint Studio:
```
The article is necessary because "plugin" is there. Or you could just remove the word "plugin" if that makes more sense.

Reply:
While I hate the wordiness, "the APIkit for OData v2 plugin" (an Anypoint Studio thing) tries to differentiate it from "the APIKit for OData v2 module" (a Mule runtime thing).

Reply:
OK! Thanks!